.github/workflows/ass-ci.yml

```diff
@@ -0,0 +1,86 @@
+name: ASS CI
```

Comment:
Can you update this to `Docker Build` so it's in lin ewith the `ts-build.yml` file